### ISSUE TITLE
Dropping PRs

### ISSUE THREAD

**Issue by AiyionPrime** [SRAhub/MinimaxSimulator#70]
@gerion0 or @wrenger Could you tell me who'd review/merge the PRs @prohde and I opened a little over 2.5 years ago?

I'd like to get rid of them in my pending PRs.
If PRs are not welcome or this repo is no longer in use, I'd suggest archiving it, which would resolve my problem as well.

Back then I found it inspiring of the SRA to maintain the software they used in their exercises.
After watching this since 2020 though, I'm not convinced it does itself a favor in doing so anymore.

Anyway. In case there's no further notice, I'll drop my Pull-Requests on the three year mark, which would be the fourteenth of December this year and gives you a headstart of another four months.

Thanks and have a nice week
Aiyion

**Comment by gerion0** [comment]
Ping @nnamua.

**Comment by prohde** [comment]
@nnamua Thanks for merging some of the PRs. What's the issue with the remaining ones?

**Comment by AiyionPrime** [comment]
Indeed, thanks @gerion0 and @nnamua it's nice to be finally done with it :)
